org.jacoco.core.test/src/org/jacoco/core/test/validation/ControlStructuresTest.java

```diff
@@ -41,8 +41,7 @@ public void testCoverageResult() {
 		assertLine("missedelse", ICounter.NOT_COVERED);
 
 		// 4. Missed while block
-		assertLine("whilefalse", isJDKCompiler ? ICounter.FULLY_COVERED
-				: ICounter.PARTLY_COVERED, 1, 1);
```

Comment:
Anyway this case was quite interesting IMO, so maybe we should preserve it somewhere?